src/main/java/com/redhat/devtools/intellij/common/listener/TreeDoubleClickListener.java

```diff
@@ -15,7 +15,7 @@
 import com.intellij.openapi.actionSystem.DataContext;
 import com.intellij.openapi.project.Project;
 import com.intellij.ui.DoubleClickListener;
-import com.intellij.util.ui.tree.WideSelectionTreeUI;
```

Comment:
`WideSelectionTreeUI` is deprecated starting 2019.3

Reply:
I use a similar tree double click listener only using `tree.getPathForLocation`, I dont think that `tree.getPathForLocation` (which is not available in IC-2021.3 any more, `WideSelectionTreeUI` was moved to `DefaultTreeUI`) is needed.